consensus/ssz_types/src/bitfield.rs

```diff
@@ -1295,4 +1323,10 @@ mod bitlist {
             assert_eq!(bitfield.ssz_bytes_len(), bytes.len(), "i = {}", i);
         }
     }
+
+    // Ensure that the stack size of a BitList is manageable.
+    #[test]
+    fn size_of() {
+        assert_eq!(std::mem::size_of::<BitList1024>(), SMALLVEC_LEN + 24);
```

Comment:
I spent a while thinking on this one... Is this comparison used just because `BitList1024` is on-hand? :sweat_smile:

Reply:
Yeah, and all BitLists should be the same size